apis/conversion/src/index.js

```diff
@@ -23,7 +23,10 @@ const getDefaultExportPropertiesFn = (path) => {
   return undefined; // TODO: add listbox and other
 };
 
-const getExportPropertiesFnc = (qae) => {
+const getExportPropertiesFnc = (qae, convertToTable) => {
+  if (convertToTable && typeof qae.exportTableProperties === 'function') {
+    return qae.exportTableProperties;
```

Comment:
consider change names to ex:
convertToTable -> convertToViewData
exportTableProperties -> exportViewDataProperties
as it is always for viewData but could be someting that is not a table if dataViewType is changed to something else

Reply:
The first one I think we can easily change. The second one is consistent with the current setup. It is already called exportTableProperties and it is only used for view data. I agree with your reasoning, but I think in that case we should also update it in the client properly. https://github.com/qlik-trial/sense-client/blob/main/web/assets/objects/utils/properties-generator/properties-generator.js#L160

@Caele What do you think?

Reply:
hmm, don't really care about this one. Either is fine.